packages/vuetify/src/components/VField/VField.sass

```diff
@@ -102,46 +95,35 @@
 /* endregion */
 /* region FIELD */
 .v-field__field
-  align-items: center
-  display: flex
   flex: 1 0
-  height: 100%
+  min-height: var(--v-input-control-height, $input-control-height)
+  grid-area: field
   position: relative
 
-  .v-field__input
-    letter-spacing: $field-letter-spacing
-    padding-top: var(--v-field-padding-top, 0)
-    padding-bottom: var(--v-field-padding-bottom, 0)
-
 /* endregion */
 /* region AFFIXES */
+.v-field__prepend-inner
+  grid-area: prepend-inner
+
+.v-field__clearable
+  grid-area: clear
+
+.v-field__append-inner
+  grid-area: append-inner
+
+.v-field__field,
 .v-field__prepend-inner,
 .v-field__append-inner,
 .v-field__clearable
-  align-items: center
+  align-items: flex-start
   display: flex
+  padding-top: var(--v-input-padding-top, $field-control-padding)
```

Comment:
This needs to be on the input element

Reply:
What does making this change solve that's broken in this PR?